nixos/modules/programs/polybar.nix

```diff
@@ -0,0 +1,41 @@
+{ config, lib, pkgs, ... }:
+
+{
+  options.programs.polybar = with lib; {
+
+    enable = mkOption {
```

Comment:
Can `mkEnableOption` be used?

Reply:
Yes, but I wanted to use the `relatedPackages` attribute as the other programs do.

Reply:
you can merge it like

```
enable = mkEnableOption "description" // {
  relatedPackages = ...
};
```